README.md

```diff
@@ -404,6 +404,12 @@ rootCmd.Flags().StringVarP(&Region, "region", "r", "", "AWS region (required)")
 rootCmd.MarkFlagRequired("region")
 ```
 
+Pesistent flags should be marked as required using a separate function:
```

Comment:
@iwwwwwwi can you maybe summarize this in the previous paragraph about required flags.
i.e:
> Flags are optional by default. If instead you wish your command to report an error
when a flag has not been set, mark it as required.  This applies to persistent flags as well.